### DISCUSSION TITLE
Content Editor - Missing domain in read request URL

### DISCUSSION THREAD

**Discussion by akulavasu** [Sunbird-Ed/Community#695] [Sunbird ED configuration discussion] [answered]
Once a image is uploaded in the content editor, the read request URL does not have the origin domain in the URL. 
![image](https://github.com/Sunbird-Ed/Community/assets/34706681/b4e2d7e4-b4c0-4313-ac64-67ef24630c0d)

@gandham-santhosh @santhosh-tg , we had tried with files subdomain as you have suggested but it still was not resolving to the correct URl and there is no issue with extra query params as of now.
The nginx configuration doesn't seem to have any issue as the below request works
https://sunbird.ex001.in/assets/public/content/do_114001678264336384110/artifact/do_114001678264336384110_1709189241342_screenshot-2023-12-07-172607.png

Kindly help us if the domain is missing to be interpolated somewhere in the application level.

**Comment by santhosh-tg** [comment]
Hi @pallakartheekreddy this is the issue I was mentioning about. Could you please help with your inputs on which configuration they need to verify.

**Comment by pallakartheekreddy** [comment]
@santhosh-tg @akulavasu Please check and fix your content service configuration because when we upload any asset API is returning artifact URL is like below
```https://https/://ax2cel5zyviy.compat.objectstorage.ap-hyderabad-1.oraclecloud.com/contents/content/do_114001740619808768113/artifact/do_114001740619808768113_1709196853204_istockphoto-1420676204-612x612.jpg```

```https://https/://``` it is having https twice. Please fix this, then above issue will be resolved.

**Reply by akulavasu** [reply]
Thanks @pallakartheekreddy , Changed the cloud_storage_proxy_host to ax2cel5zyviy.compat.objectstorage.ap-hyderabad-1.oraclecloud.com without https and editor is working now for images. 
 After content is saved, it is not showing up in drafts or under my content. Please let know what config has to be checked for the same.

**Reply by santhosh-tg** [reply]
@akulavasu please verify if you are able to read that content that you have created. 
`curl --location 'https://DOMAIN/api/content/v1/read/DO_ID' --header 'Content-Type: application/json'`
Substitute DO_ID with  do_id number of that content. Also, make sure search-indexer job is running without any errors.

Cc:  @amitpriyadarshi @vinukumar-vs

**Reply by akulavasu** [reply]
Yeah @santhosh-tg, able to read the content with provided curl.

**Reply by santhosh-tg** [reply]
@amitpriyadarshi can you share your insights please ?

**Comment by mohitga** [comment]
@santhosh-tg  can you help here and see why the content is not visible?

**Comment by santhosh-tg** [comment]
@akulavasu Hope these issues are resolved now.  Please update and closed the thread

**Reply by akulavasu** [reply, marked as answer]
Thanks for the call @santhosh-tg and @pallakartheekreddy 
Issue is resolved now. Logstash service was stopped and therefore the contents were not getting indexed.
